src/lib/QtWidgets/QSystemTrayIcon/index.ts

```diff
@@ -0,0 +1,32 @@
+import addon from "../../utils/addon";
+import { NodeWidget } from "../QWidget";
+import { BaseWidgetEvents } from "../../core/EventWidget";
+import { NativeElement } from "../../core/Component";
+import { QIcon } from "../../QtGui/QIcon";
+
+export const QSystemTrayIconEvents = Object.freeze({
+  ...BaseWidgetEvents,
+});
+export class QSystemTrayIcon extends NodeWidget {
```

Comment:
Instead of inheriting from NodeWidget you can inherit from `EventWidget` instead here. 
I know at this point EventWidget inherits from YogaWidget (I will fix that in a follow up PR).

So to give you a bit of background.
NodeWidget is a class that contains all the methods of QWidget. 
In Qt world every widget inherits from QWidget
So similarly in NodeGUI world every widget inherits from NodeWidget.
Since QSystemTrayIcon is not a widget (ie, it doesnt inherit from QWidget in Qt world) we do not inherits from NodeWidget in nodegui world.